ccmlib/node.py

```diff
@@ -319,9 +319,10 @@ def grep_log(self, expr, filename='system.log'):
         pattern = re.compile(expr)
         with open(os.path.join(self.get_path(), 'logs', filename)) as f:
             for line in f:
-                m = pattern.search(line)
-                if m:
-                    matchings.append((line, m))
+                if 'INFO' in line or 'ERROR' in line:
```

Comment:
better: `if "DEBUG" not in line:`
(since you missed WARN and adding WARN case makes the line longer and also there
is some degree of unpredictability, so better to exclude what you know)